contrib/depends/funcs.mk

```diff
@@ -213,6 +213,14 @@ $(1): | $($(1)_cached_checksum)
 
 endef
 
+stages = fetched extracted preprocessed configured built staged postprocessed cached cached_checksum
+
+define ext_add_stages
+$(foreach stage,$(stages),
+          $(1)_$(stage): $($(1)_$(stage))
+          .PHONY: $(1)_$(stage))
+endef
+
```

Comment:
did you check whitespaces??
It is very sensitive to TAB or SPACE.

Reply:
I'm not quite sure what you want here. Can you suggest a change?

Reply:
In a Makefile, tabs precede commands. I'm guessing this is the question here.